### PR TITLE
feat: Air Quality community template

### PR DESCRIPTION
**Pull request by kristinarobinson**

### CODE REVIEW

File: airquality/README.md [hunk 1]
```diff
@@ -0,0 +1,65 @@
+Provided by: Kristina Robinson
+
+Air Quality (Ozone and Small Particulates per Million) by zipcode from the U.S. EPA AirNow.gov API.
+
+Quick Install
+
+InfluxDB UI
+
+In the InfluxDB UI, go to Settings-> Templates and enter this URL: http://raw.githubusercontent.com/influxdata.community-templates.master/airquality/airquality.yaml
```
Comment (mhall119) [inline airquality/README.md:9]: This URL doesn't look right, it should be the same as the one in the CLI instructions below

File: airquality/README.md [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+
+InfluxCLI
```
Comment (mhall119) [inline airquality/README.md:11]: Add a space between Influx and CLI

File: airquality/airquality.yml [hunk 3]
```diff
@@ -0,0 +1,226 @@
...
+          ## Destination bucket to write into.  CHANGE THIS if you want to write to a different bucket
+          bucket = "airquality"   
+          # interval at which the API is called.  air quality is only updated 1-2 times per day
+          [agent]
+          interval = "12h"
+          
+        # Read formatted metrics from one or more HTTP endpoints
+        [[inputs.http]]
+          ## One or more URLs from which to read formatted metrics
+          urls = [<CHANGE ME>]
```
Comment (mhall119) [inline airquality/airquality.yml:213]: Can you make this an envvar as well?

File: airquality/README.md [hunk 4]
```diff
@@ -0,0 +1,65 @@
...
+
+Setup Instructions
+
+Go to https://docs.airnowapi.org/account/request/ to request account access.
+Go to https://docs.airnowapi.org/forecastsbyzip/query, and build your URL as follows:
+* Enter the zip code of interest
+* Select `application/json` from the Format dropdown
+* Click `Build`
+* Copy the URL string.
+Inside InfluxDB, click Data, Telegraf and edit the airquality-telegraf-config file.  Under the `[[inputs.http]]` section, paste the url value copied from the previous step where it says "CHANGEME".
```
Comment (mhall119) [inline airquality/README.md:33]: It would be easier to make this an envvar and add an instruction to set it with the `INFLUX_` envvars below

**Review by mhall119 — CHANGES_REQUESTED** [review]

File: airquality/README.md (hunk 4 above)
Comment (kristinarobinson) [reply airquality/README.md:33]: OK, I will add this as an environment var as well.

File: airquality/airquality.yml (hunk 3 above)
Comment (kristinarobinson) [reply airquality/airquality.yml:213]: will do

**Review by mhall119 — APPROVED** [review]
Looks good now, thanks!

**Pull request merged**